modules/cache/classes/Kohana/Cache/Memcache.php

```diff
@@ -287,7 +287,9 @@ public function delete_all()
 	 */
 	public function _failed_request($hostname, $port)
 	{
-		if ( ! $this->_config['instant_death'])
+		// This is for compatability with anyone using the config
+		// variable from Koseven initail update
+		if ($this->config('instant_death') === FALSE)
```

Comment:
Updated this to use the built in config function

This will return `NULL` when the value is not set so we will pass through as the inner `instant_death` will be the deciding factor then

![image](https://user-images.githubusercontent.com/1033039/96613775-e21d0080-12ee-11eb-8b97-c04c36ae354c.png)